app/features/receive/cashu-receive-quote.ts

```diff
@@ -55,7 +30,7 @@ export type CashuReceiveQuote = {
   description?: string;
   createdAt: string;
   expiresAt: string;
-  state: 'UNPAID' | 'PAID';
+  state: 'UNPAID' | 'EXPIRED' | 'PAID' | 'COMPLETED';
```

Comment:
I changed the logic not to delete the quotes but to do state updates. We will have a cleanup cron job that will be deleting old expired and completed quotes older than 1 day or so